tests/integration/sync/it_sync_location_test.py

```diff
@@ -20,27 +21,10 @@
 """
 
 
-def test_sync_location(db_session):
-    port = free_port()
-    start_server(port,
-        locations = [
-            {
-                "id": 100,
-                "name": "Coco Bongo",
-                "code": "C",
-                "company_id": 50,
-                "country": "United States",
-                "region": "East Coast",
-                "city": "Edge City",
-                "address": "Blvd. Kukulcan Km 9.5 #30, Plaza Forum",
-                "longitude": 21.1326063,
-                "latitude": -86.7473191,
-                "type": "L",
-                "status": "open",
-                "comment": "Nightclub from a famous movie"
-            }
-        ]
-    )
+@pytest.mark.skip
```

Comment:
@vladarefiev why is this test skipped?

Reply:
@amihaiemil this test is skipped because these puzzles are not resolved yet - https://github.com/vladarefiev/timelessis/blob/48bc887089ee05e3cb4f11520901bd151de2134d/timeless/restaurants/models.py#L78

Reply:
@vladarefiev can you then add a message to ``pytest.mark.skip`` to explain the reason? So that we know when to unskip the test (a link to the puzzle/ticket would also be nice). Something like:

``@pytest.mark.skip("Skipped until ticket #xyz (corresponding to that puzzle) is fixed")``

Reply:
@amihaiemil Good idea, thanks, will do it now